front/src/applications/stdcm/utils/computeOpSchedules.ts

```diff
@@ -29,12 +29,12 @@ const computeOpSchedules = (startTime: Date, msFromStartTime: number) => {
     origin: {
       date: originDate,
       time: originTime,
-      isoArrivalTime: subtractDurationFromDate(startTime, 1800).toISOString(),
+      isoArrivalTime: new Date(subtractDurationFromDate(startTime, 1800)),
```

Comment:
`subtractDurationFromDate` already returns a date, so you just need to remove the toISOString()
```suggestion
      isoArrivalTime: subtractDurationFromDate(startTime, 1800),
```

Reply:
Done